builders/deploy/dapp.md

```diff
@@ -130,38 +142,26 @@ Finally, [generate and edit](/builders/build/customize/customizing-chain-specs/#
 
 Now, you can upload your custom raw specification file by selecting the **Custom** template and adding your JSON specification file.
 
-![Upload a custom raw specification file to the Tanssi dApp.](/images/builders/deploy/dapp/dapp-4.png)
+![Upload a custom raw specification file to the Tanssi dApp.](/images/builders/deploy/dapp/dapp-7.png)
 
 !!! note
     The size of a raw chain specifications file should not exceed 2MB.
 
-## Check Your Balances {: #check-balances }
+## Check Balances {: #check-balances }
```

Comment:
you're correct! The request tokens is only for snapchains. Since the guide is focused on snapchains I'll leave the screenshot in there and mention that for dedicated appchains they will need to manually request.